src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -1802,9 +1802,13 @@ internal static string CheckSuspiciousContent(Ast scriptBlockAst)
 
         private static class SuspiciousContentChecker
         {
-            // Based on a (bad) random number generator, but good enough
-            // for our simple needs.
-            private const uint LCG = 31;
```

Comment:
If you are going to change how the hashes are generated, you need to update the hashes in the code.